tools/ci_build/github/azure-pipelines/linux-ci-pipeline.yml

```diff
@@ -1,40 +1,8 @@
 jobs:
-- job: Linux_CI_Dev
-  pool: Linux-CPU
-  steps:
-    - template: templates/set-test-data-variables-step.yml
-
-    - task: CmdLine@2
-      displayName: 'Clean untagged docker images'
-      inputs:
-        script: |
-          docker container prune -f
-          docker image prune -f
-        workingDirectory: $(Build.BinariesDirectory)
-      continueOnError: true
-      condition: always()
-
-    - task: CmdLine@2
-      displayName: 'Download azcopy'
-      inputs:
-        script: |
-          curl -so azcopy.tar.gz -L 'https://aka.ms/downloadazcopy-v10-linux'
-          tar -zxvf azcopy.tar.gz --strip 1
-        workingDirectory: $(Build.BinariesDirectory)
-
-    - task: PythonScript@0
-      displayName: 'Download test data'
-      inputs:
-        scriptPath: '$(Build.SourcesDirectory)/tools/ci_build/github/download_test_data.py'
-        arguments: --test_data_url $(TestDataUrl)
-        pythonInterpreter: '/usr/bin/python3'
-        workingDirectory: $(Build.BinariesDirectory)
-
-    - script: 'tools/ci_build/github/linux/run_dockerbuild.sh -o ubuntu16.04 -d cpu -r $(Build.BinariesDirectory) -x "--use_mklml --use_tvm --build_wheel"'
-      displayName: 'Command Line Script'
-
-    - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
-      displayName: 'Component Detection'
-      condition: and(succeeded(), in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI'))
-
-    - template: templates/clean-agent-build-directory-step.yml
+- template: templates/win-ci.yml
```

Comment:
Maybe it need a better name? 
Why the Linux CI pipeline includes a template from win-ci.yml?

Reply:
Updated to "templates/linux-ci". Part of this PR was to test the all the individual CI pipelines at one time (the NuGet pipelines  were tested manually + passing).